urql-graphql/urql-devtools
# chore(workspace): update dependencies and bring in-line with main repo

Supersedes https://github.com/urql-graphql/urql-devtools/pull/381

TODO:

- [x] Remove CSB CI
- [x] Remove renovate
- [x] Replace `graphql-tag` with `gql` from `@urql/core`
- [x] Update Babel to latest
- [x] upgrade to node v16
- [x] renew license like the main repo
- [x] fix cosmos build, seems related to having to upgrade codemirror
- [x] update and double check visual regression snapshots

This removes CircleCI entirely due to #402, we will publish manually for the time being and move to github-actions in a follow-up

Follow ups:

- check if the chrome-manifest can conform to v3
- move to github-actions for publishing
- update electron
- upgrade to webpack v5
